### DISCUSSION TITLE
极简首页推荐页能否支持切换App端推荐算法

### DISCUSSION THREAD

**Discussion by nvtijuewang** [the1812/Bilibili-Evolved#4742] [功能建议 / Ideas]
看到有其他插件有此功能 但与bilibilievolved并不兼容

**Comment by the1812** [comment]
![Snipaste_2024-05-25_10-02-27](https://github.com/the1812/Bilibili-Evolved/assets/26504152/d244d002-14dd-461f-8e12-e321f0998be5)
